clear-containers-image/update_image.sh

```diff
@@ -11,8 +11,14 @@ source ../versions.txt
 source ../scripts/pkglib.sh
 
 # get versions from clearcontainers/runtime repository
-runtime_versions_url="https://raw.githubusercontent.com/clearcontainers/runtime/master/versions.txt"
-source <(curl -sL "${runtime_versions_url}")
+latest_runtime_tag=$(curl -s https://github.com/clearcontainers/runtime/releases/latest | cut -d / -f8 | cut -d '"' -f1)
+if [ -n "$latest_runtime_tag" ] && [[ "$latest_runtime_tag" =~ ^[0-9]+\.[0-9]{0,2}+\.[0-9]{0,3}$  ]]; then
```

Comment:
This pattern won't match all valid semver versions (https://semver.org/). I think the most pragmatic fix here is simply to drop the trailing `$` anchor so this just matches on a valid prefix:

```
$ latest_runtime_tag=3.0.17
$ [[ "$latest_runtime_tag" =~ ^[0-9]+\.[0-9]{0,2}+\.[0-9]{0,3}  ]] && echo ok
ok
$ latest_runtime_tag=99.99.999-1.2.3.alpha.foo+git03b38063b27b960e26d3cfab92ede6aaf26f65b3.wibble.9.8.7foo
[[ "$latest_runtime_tag" =~ ^[0-9]+\.[0-9]{0,2}+\.[0-9]{0,3}  ]] && echo ok
ok
```